syntax = "proto3";

package MvdS.ksvc;

message KeyRing {
  string name = 1;
};
